<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21699" started-at="2018-07-26T16:44:52Z" finished-at="2018-07-26T16:45:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21699" started-at="2018-07-26T16:44:52Z" finished-at="2018-07-26T16:45:14Z">
      <class name="flipkart.tests.TestSearching">
        <test-method status="PASS" signature="testSearch()[pri:0, instance:flipkart.tests.TestSearching@eafc191]" name="testSearch" duration-ms="9945" started-at="2018-07-26T16:44:52Z" finished-at="2018-07-26T16:45:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSearch -->
        <test-method status="FAIL" signature="testSearchFails()[pri:0, instance:flipkart.tests.TestSearching@eafc191]" name="testSearchFails" duration-ms="9621" started-at="2018-07-26T16:45:02Z" finished-at="2018-07-26T16:45:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [FiltersFails] but found [Filters]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [FiltersFails] but found [Filters]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at flipkart.tests.TestSearching.testSearchFails(TestSearching.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSearchFails -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:flipkart.tests.TestSearching@eafc191]" name="afterTest" is-config="true" duration-ms="2108" started-at="2018-07-26T16:45:12Z" finished-at="2018-07-26T16:45:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- flipkart.tests.TestSearching -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
